csv_joiner_tests.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+from csv_joiner import Parser
+
+
+class CSVJoinerTest(unittest.TestCase):
+    def test_parser(self):
+        input_data = ['314,752,81,""vcbc""', '43.03,0.47,2076.45,22884',
+                        '22893,29672,ios,10.1,"iPhone9,3",3', '317,222,29,"""."',
+                        '312,102,32,"r,,,""bb"', '314,752,81,""vcbc""']
+
+        expected_output = [['314', '752', '81', '""vcbc""'], ['43.03', '0.47', '2076.45', '22884'],
+                           ['22893', '29672', 'ios', '10.1', '"iPhone9,3"', '3'], ['317', '222', '29', '"""."'],
+                           ['312', '102', '32', '"r,,,""bb"'], ['314', '752', '81', '""vcbc""']]
```

Comment:
Writing the lines one by one (or even as a single string with splitting them by a new line at the end) could help make it more easy to scan for people